vulkaninfo/vulkaninfo.h

```diff
@@ -666,23 +729,27 @@ struct AppInstance {
 
         VkResult err = dll.fp_vkCreateInstance(&inst_info, nullptr, &instance);
         if (err == VK_ERROR_INCOMPATIBLE_DRIVER) {
-            std::cerr << "Cannot create Vulkan instance.\n";
-            std::cerr << "This problem is often caused by a faulty installation of the Vulkan driver or attempting to use a GPU "
-                         "that does not support Vulkan.\n";
+            std::cerr << "Cannot create " API_NAME " instance.\n";
+            std::cerr << "This problem is often caused by a faulty installation of the " API_NAME " driver or attempting to use a GPU "
+                         "that does not support " API_NAME ".\n";
             THROW_VK_ERR("vkCreateInstance", err);
         } else if (err) {
             THROW_VK_ERR("vkCreateInstance", err);
         }
         ext_funcs.LoadInstanceExtensionDispatchPointers(dll.fp_vkGetInstanceProcAddr, instance);
 
+#ifndef VULKANSC
         err = ext_funcs.vkCreateDebugReportCallbackEXT(instance, &dbg_info, nullptr, &debug_callback);
```

Comment:
dbg_info is still unconditionally chained into VkInstanceCreateInfo above here - this is an unknown structure in VKSC.